AEXS/aex-141.md

```diff
@@ -243,14 +244,31 @@ entrypoint is_approved_for_all(owner: address, operator: address) : bool
 
 ```sophia
 datatype event 
-        = Transfer(address, address, int)
+        = Mint(address, int)
+        | Transfer(address, address, int)
```

Comment:
Burn to be added

Reply:
same here, `Burn` event only relevant in `burnable` extension. not part of default interface